install.js

```diff
@@ -0,0 +1,8 @@
+// F5 -> npm start -> npm prestart -> this script.
+// Runs `npm install` only if needed. Otherwise skips (and saves time).
+
+const fs = require('fs')
```

Comment:
Where are all the semicolons.  Make this valid TypeScript

Reply:
Technically valid with or without semicolons and in TypeScript or JavaScript. We change the linter to semi-colons but that would be more easily done in a separate PR.

Note: This script is intentionally not Typescript as it is targeted to run in NodeJS.